src/app/pokemon-list/services/pokemon.service.ts

```diff
@@ -7,9 +7,9 @@ import { Pokemon } from 'src/app/core/models/pokemon.model';
   providedIn: 'root',
 })
 export class PokemonService {
-  baseUrl = 'https://pokeapi.co/api/v2/';
-  countPokemons$ = new BehaviorSubject<number>(0);
-  countPokemons$$ = this.countPokemons$.asObservable();
+  private readonly baseUrl = 'https://pokeapi.co/api/v2/';
+  private countPokemons$ = new BehaviorSubject<number>(0);
+  public countPokemons$$ = this.countPokemons$.asObservable();
```

Comment:
Do we really need this?

Reply:
> Do we really need this?

Are you asking about access modifiers? due to lack of experience, I do not understand exactly where to explicitly specify access modifiers and where not. I need you to give me an instruction.

Reply:
perhaps for this application, the isolation of BehaviorSubject from external code is redundant
c4b36d4